tests/std/tests/P1208R6_source_location/test.cpp

```diff
@@ -143,10 +164,19 @@ constexpr void sub_member_test() {
     } else
 #endif // ^^^ workaround ^^^
     {
+#ifdef __EDG__
+        assert(s.x.loc.function_name() == "void sub_member_test()"sv);
+#else // ^^^ EDG / Other vvv
         assert(s.x.loc.function_name() == "void __cdecl sub_member_test(void)"sv);
+#endif // ^^^ Other ^^^
     }
 #else // ^^^ detailed / basic vvv
-    assert(s.x.loc.function_name() == "sub_member_test"sv);
+#if !defined(__clang__) && !defined(__EDG__) // TRANSITION, VSO-1285783
+    if (is_constant_evaluated()) {
+        assert(s.x.loc.function_name() == "main"sv);
+    } else
+#endif // ^^^ workaround ^^^
+        assert(s.x.loc.function_name() == "sub_member_test"sv);

```

Comment:
Ditto "fully braced":
```suggestion
    {
        assert(s.x.loc.function_name() == "sub_member_test"sv);
    }
```